app/presenters/pafs_core/camc3_presenter.rb

```diff
@@ -57,6 +57,42 @@ def non_residential_properties
     end
   end
 
+  def households_at_reduced_risk_2040
```

Comment:
As per previous comment - can we avoid having a year in method and attribute names also?